client/src/app/submission/submission.component.ts

```diff
@@ -198,6 +224,10 @@ export class SubmissionComponent implements OnInit, OnDestroy {
         if (this.submissionEnvelope) {
           this.getSubmissionErrors();
           this.getSubmissionManifest();
+          this.getValidationSummary();
+
+          // TODO: move this polling into data source
```

Comment:
clean up comment? i think polling is already in datasource

Reply:
No, this is referring to the polling that is done on the submission level, not entity level. We would need to refactor Submission Component to use the new PaginatedDataSource

Reply:
ah I see what you mean